src/main/java/be/seeseemelk/mockbukkit/entity/SlimeMock.java

```diff
@@ -0,0 +1,65 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Slime;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+public class SlimeMock extends MobMock implements Slime
+{
+
+	private int size = 1;
+	private boolean canWander = true;
+
+	/**
+	 * Constructs a new {@link SlimeMock} on the provided {@link ServerMock} with a specified {@link UUID}.
+	 *
+	 * @param server The server to create the entity on.
+	 * @param uuid   The UUID of the entity.
+	 */
+	public SlimeMock(@NotNull ServerMock server, @NotNull UUID uuid)
+	{
+		super(server, uuid);
+	}
+
+	@Override
+	public int getSize()
+	{
+		return this.size;
+	}
+
+	@Override
+	public void setSize(int sz)
+	{
+		if (sz < 1)
+		{
+			throw new IllegalArgumentException("Size cannot be less than 1");
+		}
+		if (sz > 127)
+		{
+			throw new IllegalArgumentException("Size cannot be greater than 127");
+		}
+		this.size = sz;
```

Comment:
By some reason the health should also be set to the max health whenever the setSize operation is made (if the entity is alive)

Reply:
The max health should also b set to size*size

Reply:
I didn't quite understand, the function only allows you to set the size of the slime, unless it also affects its health?

Reply:
Yes, it also affects the health, look at the net.minecraft.world.entity.monster.Slime#setSize(int, boolean) method

Reply:
Done in commit 4efdeb1 , I leave the exeptions related to this conversation: https://github.com/MockBukkit/MockBukkit/pull/871#discussion_r1344178260